packages/mongodb-memory-server-core/src/util/MongoBinaryDownloadUrl.ts

```diff
@@ -38,13 +38,16 @@ export class MongoBinaryDownloadUrl {
     const archive = await this.getArchiveName();
     log(`Using "${archive}" as the Archive String`);
 
-    const downloadUrl = resolveConfig('DOWNLOAD_URL');
+    const downloadUrl = resolveConfig(ResolveConfigVariables.DOWNLOAD_URL);
```

Comment:
For me looks overweighted. The old variant was better.

PS. Feel free to leave your new variant as is. It's not a big problem.

Reply:
i just made it so that there cannot be any spelling mistakes when using in multiple places, or having to rename / change the value to something to have it automatically in all places of this repo & for the user to easily change it

Reply:
For this reason, we can use literal union type:
```ts
resolveConfig(name: 'DOWNLOAD_URL' | '...' | '...' | ...)
```
TS will control us for spelling mistakes. And renaming with TS also won't be a big deal.

Reply:
> we can use literal union type

yes, but they wont be available at runtime (not that i would currently know on why it would be useful)